src/layers/point-layer/point-layer.js

```diff
@@ -264,7 +265,28 @@ export default class PointLayer extends Layer {
             colorScale: this.config.colorScale
           }
         }
-      })
+      }),
+      // text label layer
+      ...(this.config.textLabel.field
+        ? [
+            new TextLayer({
+              id: `${this.id}-label`,
+              data: data.data,
+              getPosition: data.getPosition,
+              getSize: this.config.textLabel.size,
+              getTextAnchor: this.config.textLabel.anchor,
+              getText: d => String(d.data[this.config.textLabel.field.tableFieldIndex - 1]),
+              getColor: d => this.config.textLabel.color,
```

Comment:
add some padding here?
```
getPixelOffset: [
  textLabel.anchor === 'start' ? 10 : textLabel.anchor === 'end' ? -10 : 0,
  textLabel.anchor === 'middle' ? 10 : 0
] 

```

Reply:
I thought about it, too. How about adding 'Horizontal/Vertical offset' sliders, so users can control the offset (x,y) to the point?

Reply:
or just call it offset, and we will be smart about it